src/features/import-maps.js

```diff
@@ -50,10 +53,11 @@ function processScripts () {
 }
 
 function extendImportMap (importMap, newMapText, newMapUrl) {
+  var newMap = {};
   try {
-    var newMap = JSON.parse(newMapText);
+    newMap = JSON.parse(newMapText);
   } catch (err) {
-    throw Error(process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON"));
+    console.error((process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON")) + "\n" + newMapText)
```

Comment:
Again maybe lets keep the `Error` instance, with a second `console.error` log for the map text.

```suggestion
    console.error(Error(process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON")));
    console.error(newMapText);
```

Hmm, now I'm tempted to think it could be pretty sweet to include just the appropriate line number and "frame" of the JSON causing the issue... that would be a massive help when debugging.

We could only include this in the SystemJS build for a nicer debugging experiene, removing it in the production build.

But not for this PR certainly...!

Reply:
Yes, more detailed parsing error would definitely be nice. But now we at least have the json that caused it ;)